src/types.ts

```diff
@@ -709,7 +709,6 @@ export interface RequestInvoiceArgs {
 export type Transaction = {
   amount?: string;
   boostagram?: Invoice["boostagram"];
-  badges?: Badge[];
```

Comment:
This has never been in use, so I deleted it.

Reply:
thanks for noticing it